fiftyone/core/models.py

```diff
@@ -72,7 +74,8 @@ def apply_model(
 
     Args:
         samples: a :class:`fiftyone.core.collections.SampleCollection`
-        model: a :class:`Model` or :class:`flash:flash.core.model.Task`
+        model: a :class:`Model` or :class:`flash:flash.core.model.Task` or
```

Comment:
@jacobmarks can you also update the docstring of `SampleCollection.apply_model()` to match?

Reply:
Yep :) Done!